templates/includes/sidebar.html

```diff
@@ -1,4 +1,17 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <p>Username:
+      <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
```

Comment:
The URL for `driver-detail` uses `pk=user.id`. Ensure that the `user.id` corresponds to the primary key of the `Driver` model. If the `Driver` model uses a different field for the primary key, this might lead to incorrect URL generation.